docs/docs/visual-testing-with-storybook.md

```diff
@@ -102,8 +102,8 @@ module.exports = ({ config }) => {
 > When using TypeScript, add this rule:
 
 ```js:title=.storybook/webpack.config.js
-
-config.module.rules.push({
+module.exports = ({ config }) => {
```

Comment:
This change definitely makes sense as it's a bit weird that we close the config.

When reading through the whole tutorial this doesn't seem 100% correct. I'm a bit stuck on the wording "When using TypeScript, add this rule:". We want to add a rule, not start a new config. I'm not 100% sure how to properly make this better. We might want to look at the diff type.

My suggestion: (thanks to @pieh for suggesting diff)
````

```diff:title=.storybook/webpack.config.js
// Prefer Gatsby ES6 entrypoint (module) over commonjs (main) entrypoint
config.resolve.mainFields = ["browser", "module", "main"]
+
+   config.module.rules.push({
+     test: /\.(ts|tsx)$/,
+     loader: require.resolve('babel-loader'),
+     options: {
+       presets: [['react-app', {flow: false, typescript: true}]],
+       plugins: [
+         require.resolve('@babel/plugin-proposal-class-properties'),
+         // use babel-plugin-remove-graphql-queries to remove static queries from components when rendering in storybook
+         require.resolve('babel-plugin-remove-graphql-queries'),
+       ],
+     },
+   });
+
+   config.resolve.extensions.push('.ts', '.tsx');
+
return config
```
````

Demo:
![image](https://user-images.githubusercontent.com/1120926/62051648-1735c280-b214-11e9-8d95-542eaf7755d8.png)

Reply:
Yes that makes sense